lib/doorkeeper/models/concerns/expirable.rb

```diff
@@ -2,12 +2,12 @@ module Doorkeeper
   module Models
     module Expirable
       def expired?
-        expires_in && Time.now > expired_time
+        expires_in && Time.now.utc > expired_time
       end
 
       def expires_in_seconds
         return nil if expires_in.nil?
-        expires = (created_at + expires_in.seconds) - Time.now
+        expires = (created_at + expires_in.seconds) - Time.now.utc
```

Comment:
> How does the .seconds method work outside of rails? It's added to the Numeric module:

Pretty sure it does not, I'm surprised Grape users didn't submit related issues.